access-db.js

```diff
@@ -434,6 +434,42 @@ async function takeAdminPriviledge(userEmail) {
     }
 }
 
+async function deleteCompany(companyName, companyStreet, companyCity, companyProvinceState, companyCountry, companyPostalZipCode, companyEmail, adminEmail) {
```

Comment:
Unfortunately we can't use this logic since we are deleting an entire json file, not just modifying it: see this tutorial https://docs.microsoft.com/en-us/azure/cosmos-db/sql/sql-api-nodejs-get-started#DeleteItem. 

Basically the logic should be:
1. query for the specific company, probably by company email (SELECT c.id, c.companyEmail WHERE c.companyEmail = companyEmail)
2. throw the id and partition key into the code from the tutorial

Reply:
right now the logic is to delete all the companies from the database and reupload them all without the one we want to delete, thats going to cost us a bunch of RUs and possibly delete all our data

Reply:
Is that not what they do on lines 467 and 468?

Reply:
I see it now Skyla is 100% correct, the query currently returns all the companies and seems to use a for loop to look for the specific company. We can straight up in the query search for the company to delete and then will only get one result, so will no longer need to loop through a list of companies.